src/providers/export/azureCustomVision.test.ts

```diff
@@ -46,15 +71,15 @@ describe("Azure Custom Vision Export Provider", () => {
         const provider = new AzureCustomVisionProvider(testProject, testProject.exportFormat.providerOptions);
         const newOptions = await provider.save(testProject.exportFormat);
 
-        expect(axios.post).toBeCalledWith(
-            // tslint:disable-next-line:max-line-length
-            expect.stringContaining("https://southcentralus.api.cognitive.microsoft.com/customvision/v2.2/Training/projects?"),
-            null,
-            expect.objectContaining({
-                headers: {
-                    "Training-key": customVisionOptions.apiKey,
-                },
-            }));
+        const customVisionProject: IAzureCustomVisionProject = {
```

Comment:
Same as above. I don't know for sure if we'll be re-using these test objects, but it would be nice to have them in place as the project grows and more tests are written